.github/workflows/cml.yaml

```
@@ -1,23 +1,29 @@
 name: CML Report
-on: push
+on: [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.head.sha || github.sha }}
+  cancel-in-progress: true
 jobs:
   run:
-    runs-on: [ubuntu-latest]
+    continue-on-error: ${{ github.ref_name != 'main' }}
```

Q: what does it give us - same question as above pretty much ...

---

Agreed, this is essentially saying even if this job fails report it as successful unless it runs against the main branch.

In theory, it could be nice for a user as it would reduce notifications to their email, but there are better ways to solve that problem IMO. This not how you normally see this feature used and I don't think it should be in a getting started type example.

---

no, it's saying even if this job is cancelled, don't report it at all.

Otherwise you [get an error](https://github.com/iterative/example-get-started/actions/runs/3301197176)

---

Mind you; unwanted side-effect: ignores non-pr-branch-push failures.

---

also I guess this is better:

```suggestion
    continue-on-error: ${{ github.ref_name != github.event.repository.default_branch }}
```

(seen in https://github.com/iterative/link-check/pull/18#discussion_r933000447)